packages/main/src/StepInput.js

```diff
@@ -531,6 +541,8 @@ class StepInput extends UI5Element {
 			this._validate();
 			this._setButtonState();
 			this._focused = true;
+			this.focused = true;
+			this.setAttribute("focused", "");
```

Comment:
It is strange, on the first line you add focused attribute, on the second one you remove it?

Reply:
done